Tasks/AppStoreRelease/task.json

```diff
@@ -294,7 +319,7 @@
             "defaultValue": "",
             "helpMarkDown": "Optionally specify the group(s) of external testers this build should be distributed to. To specify multiple groups, separate group names by commas e.g. 'External Beta Testers,TestVendors'. If not specified the default 'External Testers' is used.",
```

Comment:
Please add some details - that it is required for 'distribute only'

Reply:
Discussed it with Anatolii. This is not required for distribute command.
Updated info in helpMarkDown.